docs/updating.rst

```diff
@@ -1,36 +1,63 @@
-===============================================================
-How to update the Atomic Developer Bundle to the latest version
-===============================================================
+===========================================================
+How to update Atomic Developer Bundle to the latest version
+===========================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
 
-  1. Check if a newer version of the box is available use this command::
+-------------
+Prerequisites
+-------------
 
-      # vagrant box outdated
+1. If ADB is running, run the `vagrant halt` command::
 
-  2. If a newer version is available, the output should look like this::
+    # vagrant halt
 
-        # vagrant box outdated
+2. Delete the virtual machine with the `vagrant destroy` command. This
+   action will also delete any data or configuration in your ADB that is not saved
+   on your host machine::
 
-     Checking if box 'projectatomic/adb' is up to date...
-     A newer version of the box 'projectatomic/adb-testing' is available! You
-     currently have version '1.3.1'. The latest is version '1.3.2'. Run ``vagrant
-     box update`` to update.
+    # vagrant destroy
 
-  3. If you see the above output, update the ADB Vagrant image with this
-     command::
+3. If you are updating ADB on a Windows machine, run the following command to
+   delete the `.vagrant.d` directory::
 
-      # vagrant box update
+     # rm -rf C:/Users/<user_name>/.vagrant.d/
```

Comment:
I'd like tosee the response to https://github.com/projectatomic/adb-atomic-developer-bundle/issues/500#issuecomment-244985560 incorporated here - this feels like a cloud.centos edge case